newIDE/app/src/Debugger/index.js

```diff
@@ -188,6 +210,13 @@ export default class Debugger extends React.Component<Props, State> {
       }));
     } else if (data.command === 'hotReloader.logs') {
       // Nothing to do.
+    } else if (data.command === 'console.log') {
```

Comment:
nit: `console.logs`
(this means you're receiving the logs from the console, not that you're instructing the console to log)

Reply:
It is not sending a list of logs though, it really is just asking to log one message

Reply:
A right I was mistaken! Looking good then,